### PR TITLE
[zh-cn]: sync with events

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

**Comment by fuchunhui** [comment]
https://github.com/mdn/content/pull/34101

File: files/zh-cn/web/events/index.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 ---
 
 [事件](/zh-CN/docs/Learn/JavaScript/Building_blocks/Events)的触发用于通知“对变化感兴趣”的代码以影响代码的执行。这些事件可能来自用户的交互（例如：使用鼠标或调整窗口大小）、底层环境状态的改变（例如：电池电量不足或操作系统的媒体事件）和其他的情况。
 
 每个事件都使用继承自 {{domxref("Event")}} 接口的对象来表示，可以包括额外的自定义成员属性及函数，以获取事件发生时相关的更多信息。本文为每个事件都提供了一个表格（靠近顶部的区域），其中包含了指向相关事件接口的链接和其他相关的信息。[事件 > 基于 `Event` 的接口](/zh-CN/docs/Web/API/Event#基于_event_的接口)中完整地列出了不同的事件类型。
 
-本文提供了你可能感兴趣的主要事件类型（动画、剪切板、worker，等等）的索引，以及实现这些事件的主要的类。最后是所有提供了文档的事件列表。
+本文提供了你可能感兴趣的主要事件类型（动画、剪切板、worker 等等）的索引，以及实现这些事件的主要的类。
 
 > **备注：** 此页面列出了你会在 web 上遇到的绝大多数常见事件。如果你正在查找一个这里没有列出的事件，请尝试在 MDN 上搜索它的名字、主题或关联的规范。
```
Comment (PassionPenguin) [inline files/zh-cn/web/events/index.md:15]:
```suggestion
```

**Review by PassionPenguin — CHANGES_REQUESTED** [review]
calls for a removal for this note, as it's also written here for the **deleted events list** (mdn/content#31220)

File: files/zh-cn/web/events/index.md (hunk 1 above)
Comment (fuchunhui) [reply files/zh-cn/web/events/index.md]: thanks

**Comment by PassionPenguin** [comment]
@fuchunhui you may now reopen the PR and @yin1999 merge it given upstream's merged :)

**Comment by fuchunhui** [comment]
> @fuchunhui you may now reopen the PR and @yin1999 merge it given upstream's merged :)

thanks for your reminder, I haven't done this before.

**Comment by yin1999** [comment]
I believe @PassionPenguin want you to mark this PR as "Ready for review". You can see this button below the CI tests list.

image

If you are thinking this PR is ready for review, please do so. And I'm willing to review this.

**Comment by PassionPenguin** [comment]
> I believe @PassionPenguin want you to mark this PR as "Ready for review". You can see this button below the CI tests list.
> 
> image
> 
> If you are thinking this PR is ready for review, please do so. And I'm willing to review this.

oops..sorry for the incorrect action name mentioned above. 😢

**Comment by fuchunhui** [comment]
> I believe @PassionPenguin want you to mark this PR as "Ready for review". You can see this button below the CI tests list.
> 
> image
> 
> If you are thinking this PR is ready for review, please do so. And I'm willing to review this.

thanks, it works.

File: files/zh-cn/web/api/event/index.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 slug: Web/API/Event
+l10n:
+  sourceCommit: 15f0b5552bc9c2ea1f32b0cd5ee840a7d43c887e
 ---
 
-{{APIRef("DOM")}}
+{{APIRef("DOM")}}{{AvailableInWorkers}}
 
-**`Event`** 接口表示在 DOM 中出现的事件。
+**`Event`** 接口表示在 [`EventTarget`](/zh-CN/docs/Web/API/EventTarget) 中出现的事件。
```
Comment (yin1999) [inline files/zh-cn/web/api/event/index.md:10]:
```suggestion
**`Event`** 接口表示在 [`EventTarget`](/zh-CN/docs/Web/API/EventTarget) 上出现的事件。
```

File: files/zh-cn/web/events/index.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 ---
 title: 事件参考
 slug: Web/Events
+l10n:
+  sourceCommit: 19e30e4787fff9074a50f16947dd2f6e9b7691c5
```
Comment (yin1999) [inline files/zh-cn/web/events/index.md:5]:
```suggestion
  sourceCommit: 4ca4fa3d9d1de5601b406b13dc75aff30709c66f
```

File: files/zh-cn/web/events/index.md [hunk 4]
```diff
@@ -521,15 +489,9 @@ slug: Web/Events
...
+          与任何性能相关的规范分组为
           <a href="/zh-CN/docs/Web/API/Performance_API"
-            >High Resolution Time API</a
-          >、<a href="/zh-CN/docs/Web/API/Performance_API/Navigation_timing"
-            >Navigation Timing API</a
-          >、<a href="/zh-CN/docs/Web/API/Performance_API/User_timing">User Timing API</a>
-          和
-          <a href="/zh-CN/docs/Web/API/Performance_API/Resource_timing"
-            >Resource Timing API</a
+            >Performance APIs</a
```
Comment (yin1999) [inline files/zh-cn/web/events/index.md:494]:
```suggestion
            >Performance API</a
```

File: files/zh-cn/web/events/index.md [hunk 5]
```diff
@@ -521,15 +489,9 @@ slug: Web/Events
...
           > 相关的事件。
```
Comment (yin1999) [inline files/zh-cn/web/events/index.md:495]:
```suggestion
          > 的相关事件。
```

File: files/zh-cn/web/events/index.md [hunk 6]
```diff
@@ -434,7 +402,7 @@ slug: Web/Events
           ><code>VideoTrackList</code></a
         >、<a href="/zh-CN/docs/Web/API/HTMLTrackElement#events"
           ><code>HTMLTrackElement</code></a
-        >、<a href="/zh-CN/docs/Web/API/OfflineAudioContext#events"
+        >、<a href="/zh-CN/docs/Web/API/OfflineAudioContext#事件"
           ><code>OfflineAudioContext</code></a
         >、<a href="/zh-CN/docs/Web/API/TextTrack#events"><code>TextTrack</code></a
         >、<a href="/zh-CN/docs/Web/API/TextTrackList#events"
```
Comment (yin1999) [inline files/zh-cn/web/events/index.md:408]: 410、411 行需要更新一下

**Review by yin1999 — APPROVED** [review]

**Pull request merged**